src/lambda/activation.ts

```diff
@@ -57,6 +58,7 @@ export async function activate(context: ExtContext): Promise<void> {
             'aws.copyLambdaUrl',
             async (node: LambdaFunctionNode) => await copyLambdaUrl(node, new DefaultLambdaClient(node.regionCode))
         ),
+        Commands.register('aws.uploadToAws', async (arg: vscode.Uri) => await uploadToAwsCommand(arg)),
```

Comment:
this is not a lambda-specific command, is there somewhere more relevant it could live?